hack/build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+set -o nounset
```

Comment:
Maybe it's better to use `set -euo pipefail` instead.

Reply:
I'd prefer the following one, it's more readable.
```
set -o nounset
set -o errexit
set -o pipefail
```